mlctl/plugins/utils.py

```diff
@@ -8,3 +8,14 @@ def parse_config(config, parameters):
             if not kwargs.get(param):
                 kwargs[param] = None
         return kwargs
+
+def run_subprocess(command):
```

Comment:
Didn't we move away from the `subprocess` method of invoking other python commands?

Reply:
This is used for the azure library. We moved away from subprocess for setup tools